pkg/trace/pb/agent_payload.proto

```diff
@@ -0,0 +1,21 @@
+// protoc -I. -I$GOPATH/src --gogofaster_out=. span.proto tracer_payload.proto agent_payload.proto
+
+syntax = "proto3";
+
+package pb;
+
+import "tracer_payload.proto";
+
+// AgentPayload represents payload the agent sends to the intake.
+message AgentPayload {
+        // hostName specifies hostname of where the agent is running.
+        string hostName = 1;
+        // env specifies `env` set in agent configuration.
+        string env = 2;
+        // tracerPayloads specifies list of the payloads received from tracers.
+        repeated TracerPayload tracerPayloads = 5;
+        // tags specifies tags common in all `tracerPayloads`.
+        map<string, string> tags = 6;
+        // agentVersion specifies version of the agent.
+        string agentVersion = 7;
```

Comment:
Could you add errorTPS and targetTPS values if they are different then the default (10). This would be of great value to track customers that have updated it

Reply:
This would be the values to add: https://github.com/DataDog/datadog-agent/blob/main/pkg/trace/config/config.go#L72-L73

Reply:
Got it, will add.